### PR TITLE
#132 Added some comments for API client

### PR DESCRIPTION
**Pull request by harshilsharma**
#### Summary
Added some comments for API client

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Added API documentation (required for all new APIs)

### CODE REVIEW

**Comment by harshilsharma** [comment]
@hanzei I've added the doc on `client4.go`. One API doc remains for `GetUserTermsOfService`, which I'll add once we are final on it on the API reference PR.

File: model/client4.go [hunk 1]
```diff
@@ -3919,7 +3919,8 @@ func (c *Client4) GetRedirectLocation(urlParam, etag string) (string, *Response)
 	return MapFromJson(r.Body)["location"], BuildResponse(r)
 }
 
-func (c *Client4) RegisteTermsOfServiceAction(userId, termsOfServiceId string, accepted bool) (*bool, *Response) {
+// Saves action performed by a user against a specific terms of service.
```
Comment (hanzei) [inline model/client4.go]: The method name should be prepended to the comment. This is the idiomatic way to write docs in go.

```suggestion
// RegisterTermsOfServiceAction saves an action performed by a user against a specific terms of service.
```

File: model/client4.go [hunk 2]
```diff
@@ -3930,6 +3931,7 @@ func (c *Client4) RegisteTermsOfServiceAction(userId, termsOfServiceId string, a
 	return NewBool(CheckStatusOK(r)), BuildResponse(r)
 }
 
+// Fetches the latest terms of service
```
Comment (hanzei) [inline model/client4.go]:
```suggestion
// GetTermsOfService fetches the latest terms of service.
```

File: model/client4.go [hunk 3]
```diff
@@ -3950,6 +3952,7 @@ func (c *Client4) GetUserTermsOfService(userId, etag string) (*UserTermsOfServic
 	return UserTermsOfServiceFromJson(r.Body), BuildResponse(r)
 }
 
+// Creates new terms of service.
```
Comment (hanzei) [inline model/client4.go]:
```suggestion
// CreateTermsOfService creates a new terms of service.
```

**Comment by hanzei** [comment]
@harshilsharma Please take a look at my comments. Otherwise :+1:

File: model/client4.go (hunk 1 above)
Comment (harshilsharma63) [reply model/client4.go]: @hanzei yes, of course. Might have been low of coffee :P Fixed everywhere.

File: model/client4.go (hunk 2 above)
Comment (harshilsharma63) [reply model/client4.go]: Fixed.

File: model/client4.go (hunk 3 above)
Comment (harshilsharma63) [reply model/client4.go]: Fixed.

**Comment by harshilsharma63** [comment]
Fixed!

File: model/client4.go [hunk 4]
```diff
@@ -3939,7 +3941,7 @@ func (c *Client4) GetTermsOfService(etag string) (*TermsOfService, *Response) {
 	defer closeBody(r)
 	return TermsOfServiceFromJson(r.Body), BuildResponse(r)
 }
-
```
Comment (hanzei) [inline model/client4.go]: The build failed because a linebreak is missing here. Run `make gofmt` to check if you fixed it.

Comment (harshilsharma) [reply model/client4.go]: Done.

**Pull request merged**